lib/ecs_deploy/scheduled_task.rb

```diff
@@ -9,7 +9,7 @@ class PutTargetsFailure < StandardError; end
 
     def initialize(
       cluster:, rule_name:, schedule_expression:, enabled: true, description: nil, target_id: nil,
-      task_definition_name:, revision: nil, task_count: nil, role_arn:,
+      task_definition_name:, revision: nil, task_count: nil, role_arn:, network_configuration: nil, launch_type: nil, platform_version: nil, group: nil,
```

Comment:
You should also change the following code 👀 

https://github.com/reproio/ecs_deploy/blob/5c4e1df527bba981cc63332e31bd89f3eab66365/lib/ecs_deploy/capistrano.rb#L58-L71